docs/readmes/lte/s1ap_tests.md

```diff
@@ -33,19 +33,33 @@ acts as the gateway between *magma_test* and *magma_trfserver*.
 ### Gateway VM setup
 
 There are two options for setting up the gateway VM, with the difference being the magma installation
-method: it can either be installed via `make` or from debian packages, the latter being the
-method by which magma is usually deployed. For everyday development, the `make` installation is
-recommended, while the debian installation is useful for testing packages before release.
+method: it can either be installed by directly compiling the services or from debian packages, the
+latter being the method by which magma is usually deployed. For everyday development, directly
+compiling the services is recommended, while the debian installation is useful for testing packages
+before release.
 
 > **Warning**: These two VMs use the same network configuration, so one must only run one of them
 at a time.
 
-#### Make installation
+#### Directly compiling services
 
-Spin up and provision the gateway VM, then make and start its services:
+Spin up and provision the gateway VM. From `magma/lte/gateway` on the host machine run
+`vagrant up magma && vagrant ssh magma`.
 
-1. From `magma/lte/gateway` on the host machine: `vagrant up magma && vagrant ssh magma`
-1. Now in the gateway VM: `cd $MAGMA_ROOT/lte/gateway && make run`
+Now build the services with either make or bazel.
+
+##### Building with make
+
+In the gateway VM run `cd $MAGMA_ROOT/lte/gateway && make run`.
+
+##### Building with bazel
+
+> :warning: **Bazel based builds are still experimental but can already be used for development**
```

Comment:
- Nit-pick: A full stop at end of sentence would be nice.
- FYI: Did you know, that
  ```
  > **Warning**
  > The full text message.
  ```
  would result in a nice styling in Github.
  > **Warning**
  > The full text message.

  However, unfortunately not in Docusaurus, such that it is not applicable here.